exotica_core/src/kinematic_tree.cpp

```diff
@@ -1008,7 +1083,47 @@ void KinematicTree::SetFloatingBaseLimitsPosXYZEulerZYX(
     }
     for (int i = 0; i < 6; ++i)
     {
+        // Set velocity and acceleration to +inf
         controlled_joints_[i].lock()->joint_limits = {lower[i], upper[i]};
+        controlled_joints_[i].lock()->velocity_limit = {inf};
```

Comment:
I wouldn't do this - it implicitly modifies parameters that were not passed in! If you have set velocity and acceleration from the RobotModel, you may not want to override them and that is why you use this signature rather than another. Please remove the changes to velocity and acceleration limits from this method.